CRM/Membershipextrasimporterapi/EntityImporter/LineItem.php

```diff
@@ -230,13 +255,25 @@ private function getFinancialTypeId() {
     throw new CRM_Membershipextrasimporterapi_Exception_InvalidLineItemException('Invalid line item "Financial Type"', 200);
   }
 
+  private function getTaxAmount() {
+    if (!empty($this->rowData['line_item_tax_amount'])) {
```

Comment:
Should not we check if $rowData['line_item_tax_amount'] amount is  CRM_Utils_Type::T_MONEY?

Reply:
no, because we already have that on the API level, so it won't reach here unless that value was of type T_MONEY